maintainers/maintainer-list.nix

```diff
@@ -11378,4 +11378,14 @@
     github = "zupo";
     githubId = 311580;
   };
+  evalexpr = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
This is already in a separate commit, do you mean a separate PR?

None of the maintainers above my one are in alphabetical order FWIW but I have put my one where it belongs.